src/test/resources/vanilla-cf-stadalone.yml

```diff
@@ -17,23 +17,15 @@
 # under the License.
 #
 name: Vanilla CloudFoundry example
-location:
-    cloudfoundry:
-      user: user@mail.com
-      password: secret
-      org: secret_organization
-      endpoint: https://api.run.secret.io
-      space: development
-      address: run.secret.io
-      memory: 1024
-      disk_quota: 10
-      instances: 1
-
+location: pivotal-ws
```

Comment:
I can add a new file with the long  location description because this YAML is used by a a LiveTests which requires real pivotal-ws configuration